aries-site/src/components/feedback/Feedback.js

```diff
@@ -0,0 +1,124 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  Form,
+  Heading,
+  Layer,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { FormClose } from 'grommet-icons';
+
+export const Feedback = ({
+  children,
+  modal,
+  onChange,
+  onClose,
+  onEsc,
+  onSubmit,
+  onReset,
+  show,
+  title,
+  value,
+  isSucessful,
+}) => {
+  const size = useContext(ResponsiveContext);
+
+  let content = (
+    <Box
+      fill="vertical"
+      overflow="auto"
+      width={!['xsmall', 'small'].includes(size) ? 'medium' : undefined}
+      pad="medium"
+      flex
+    >
+      <Identifier onClose={onClose} title={title} modal={modal} />
+      <Box gap="medium" margin={{ vertical: 'small' }}>
+        <Form
+          value={value}
+          onChange={onChange}
+          onSubmit={onSubmit}
+          onReset={onReset}
+          method="post"
+          validate="submit"
+        >
+          {children}
+          <>
+            {isSucessful !== true ? (
```

Comment:
```suggestion
            {i!sSucessful ? (
```
also, typo in prop and here --> should be isSuccessful